.github/workflows/benchmark.yml

```diff
@@ -50,7 +58,7 @@ jobs:
     outputs:
       matrix: ${{ steps.set-matrix.outputs.matrix }}
     env:
-      SPARK_BENCHMARK_NUM_SPLITS: ${{ github.event.inputs.num-splits }}
+      SPARK_BENCHMARK_NUM_SPLITS: ${{ inputs.num-splits }}
```

Comment:
The change from `github.event.inputs.xxx` to `inputs.xxx` is not related to this PR and is only for the purpose of following https://github.blog/changelog/2022-06-10-github-actions-inputs-unified-across-manual-and-reusable-workflows/
<img width="714" alt="image" src="https://github.com/user-attachments/assets/404b062d-0914-4245-81bf-46757b4608db">